### ISSUE TITLE
NUnitLite updates?

### ISSUE THREAD

**Issue by Temtaime** [xamarin/xamarin-android#4895]
Hello.
Will there be an update for Xamarin.Android.NUnitLite ?
Seems that this dll is a bit outdated and doesn't support new nunit.framework attributes.
How one is supposed to run NUnit tests with Xamarin.Android(not Forms)?

Also seems that for now there're Xamarin.Android.NUnitLIte(for pure Xamarin) and NUnit.Xamarin(for Forms). Will any of them be deprecated?
Thanks for your replies.

**Comment by Temtaime** [comment]
@rprouse @CharliePoole

**Comment by CharliePoole** [comment]
@Temtaime `Xamarin.Android.NUnitLite` not published by `NUnit.Xamarin` are not published by the NUnit Project.

I created NUnitLite around 2007 as a separate framework from NUnit. It was ultimately released as NUnitLite 1.0 and was not developed further. When we created NUnit 3, we merged in the facilities of NUnitLite. One important difference is that the original NUnitLite actually was a test framework whereas the current NUnitLite component of NUnit is used __in addition to__ the NUnit framework.

I believe the Xamarin frameworks are based on a fork of the original NUnitLite taken at version 0.7, with various additions over time. For a while I had discussions with Xamarin folks about merging the products and taking the Xamarin additions into our own framework but unfortunately neither side had enough bandwidth to devote to it.
